mgr/src/main.rs

```diff
@@ -34,6 +35,11 @@ struct InstallArgs {
     /// Do not reboot after installation
     #[clap(long, action)]
     no_reboot: bool,
+
+    /// The mnemonic phrases and macaroons will automatically generate on remote server when kld first initialize.
+    /// This benefits when you own your remote server and can physically backup mnemonic phrases and macaroons without any copy through the internet
```

Comment:
// When you first initialize KLD, mnemonic phrases and macaroons will automatically be // generated on your remote server. This is advantageous if you own your remote server, // as you can physically back up your mnemonic phrases and macaroons without the need to // transmit any copies over the internet.

Reply:
```suggestion
    /// When you first initialize KLD, mnemonic phrases and macaroons will automatically be 
    /// generated on your remote server. This is advantageous if you own your remote server,
   /// as you can physically back up your mnemonic phrases and macaroons without the need to
   /// transmit any copies over the internet.
```

Reply:
These comment are added.